package.xml

```diff
@@ -28,6 +28,7 @@
   <build_depend>tf2_geometry_msgs</build_depend>
   <build_depend>tf2_msgs</build_depend>
   <build_depend>tf2_ros</build_depend>
+  <build_depend>rosconsole</build_depend>
```

Comment:
thanks for the quick fix, one tiny comment: can you place it in alphabetical order in the list ? (it looks like all the other deps are ordered alphabetically) Same below for the run_depend